src/component/Flow.tsx

```diff
@@ -385,24 +146,31 @@ export default class Flow extends React.Component<FlowProps, FlowState> {
      * existing node or on to empty space.
      */
     private onConnectorDrop(event: ConnectionEvent): boolean {
-        // we put this in a zero timeout so jsplumb doesn't swallow any stack traces
+        const { ghostNode, pendingConnection } = this.props;
+        // We put this in a zero timeout so jsplumb
+        // doesn't swallow any stack traces.
         window.setTimeout(() => {
-            if (this.state.ghost) {
-                // wire up the drag from to our ghost node
-                const dragPoint = this.pendingConnection;
-
-                this.Plumber.recalculate(this.state.ghost.uuid);
-                this.Plumber.connect(dragPoint.exitUUID, this.state.ghost.uuid);
+            if (ghostNode) {
+                // Wire up the drag from to our ghost node
+                const dragPoint = pendingConnection;
+                this.Plumber.recalculate(ghostNode.uuid);
+                this.Plumber.connect(dragPoint.exitUUID, ghostNode.uuid);
 
-                // save our position for later
+                // Save our position for later
                 const { left, top } = snapToGrid(
-                    this.ghost.ele.offsetLeft,
-                    this.ghost.ele.offsetTop
+                    this.ghost.wrappedInstance.ele.offsetLeft,
```

Comment:
`react-redux`'s `connect` method produces a new component that keeps a reference to the component it wraps in the `wrappedInstance` key. that's what we have to reference if we pass a ref to a connected component.